src/EventListener/ReadListener.php

```diff
@@ -147,6 +147,10 @@ private function getSubresourceData(Request $request, array $attributes, array $
 
         $data = $this->subresourceDataProvider->getSubresource($attributes['resource_class'], $identifiers, $attributes['subresource_context'], $attributes['subresource_operation_name']);
 
+        if (null === $data) {
+            throw new NotFoundHttpException(sprintf('Object of type "%s" with identifiers %s could not be found', $attributes['resource_class'], implode(',', $identifiers)));
```

Comment:
Can you change the message to keep consistency with this:
https://github.com/api-platform/core/blob/d4aa84f4eac58e7900b6f4d30947d3926902bb98/src/EventListener/ReadListener.php#L175